qa/common/gen_qa_model_repository

```diff
@@ -525,3 +523,66 @@ if [ $? -ne 0 ]; then
     echo -e "Failed"
     exit 1
 fi
+
+else # [[ "aarch64" != $(uname -m) ]]
```

Comment:
Is this to add support for jetson in the model gen scripts?

Reply:
Yes it's added to generate models directly on Jetson device. <br>
Jetson NVIDIA doesn't ship all the required images for `aarch64` architecture, so we workaround it by using Jetson hardware instead.

Reply:
Can you maybe add a comment on that here? Also nit I think we have been doing if else statements in bash like this:
```
if [ something ]; then
   ...
else
   ...
fi
```
https://github.com/triton-inference-server/server/blob/main/qa/L0_infer/test.sh#L71-L75

Reply:
It actually has this structure but to be able see it you need to load entire file

Reply:
In that case, is the indentation incorrect then? 
```
if [ $? -ne 0 ]; then
    echo -e "Failed"
    exit 1
fi

else # [[ "aarch64" != $(uname -m) ]]
```

is not following the convention

Reply:
It didn't follow the convention earlier either, and reason for it that wea are using `cat >> file.name<<EOF` construction to create script entries. 
By start using  the  proposed convention  for `if/else` we have to bear in mind that the indentation will be applied to the autogenerated scripts.

Reply:
Actually I still don't fully understand how this works? The if condition used to be only for skipping openvino models on ARM SBSA but looks like with your changes we will be skipping ONNX and PyTorch too? Is my understanding correct?

Reply:
I think your understanding is correct Iman, it looks like after this change we are skipping everything except the final section on aarch64. 

@mc-nv could we make this easier to read by grouping the sections into functions:
```bash
function gen_openvino() {
...
}
function gen_onnx() {
...
}
...
### MAIN ###
if [[ x86 ]]
  gen_openvino()
  gen_onnx()
  gen_tf()
  ...
else   # aarch64
  # skip Openvino on aarch64
  gen_onnx()
  gen_tf(
  ...  
fi
```

Reply:
> reason for it that wea are using cat >> file.name<<EOF construction to create script entries.

The indentation will be added to the generated script, but will it make the script no longer work as intended?

Reply:
I think its fine to exclude the indentation if we move the sections to functions. If we want the indentation for readability, but not to include it in the resulting file, we can pipe through some `sed -e "s/^\s*//"` or similar to remove leading whitespaces on each line before writing to target file